test/generate.test.js

```diff
@@ -159,6 +171,23 @@ function define (t) {
     })
   }
 
+  function verifyStandardlintInPkg (t) {
+    return new Promise((resolve, reject) => {
+      const pkgFile = path.join(workdir, 'package.json')
+
+      readFile(pkgFile, function (err, data) {
```

Comment:
Could you please use `fs.promises.readFile()`.

Reply:
sure, I just tried to follow the existing code pattern, but it's fixed with what you asked now.